### DISCUSSION TITLE
Autoadjust Output to Home Power

### DISCUSSION THREAD

**Discussion by userkai14122001** [thomluther/ha-anker-solix#84] [Ideas]
**Thank you for the great work!**

How about adjusting the battery output based on the house's power consumption to maximize the use of battery power without feeding excess power into the grid?

**Comment by thomluther** [comment]
@userkai14122001 
This is possible and you can automate it. its described here in #81

**Reply by thealkly** [reply]
I've used the battery output to even create a zero fed in automation. Based on Thomluther and others from PV Forum. If you don't change the values too often, it is working pretty good during the day. In the night I use a OpenDTU controllable Micro Inverter to control every couple of seconds the output. I've made a[ YT video ](https://youtu.be/nvjEaUTmslA) where I explain everything. Hope this helps to give you some inspiration.
